setup.py

```diff
@@ -64,6 +64,7 @@
             'pyroomacoustics.datasets',
             'pyroomacoustics.bss',
             'pyroomacoustics.denoise',
+            'pyroomacoustics.phase',
```

Comment:
Oups! I believe that I forgot to add this after adding the phase subpackage. Thanks for catching this! 👍